### PR TITLE
Update examples/storm for v1beta3

### PR DESCRIPTION
**Pull request by bcbroussard**
Small part of #6584.

### CODE REVIEW

File: examples/storm/README.md [hunk 1]
```diff
@@ -155,16 +155,16 @@ before and after creating the ReplicationController.
 
 ## tl;dr
 
-```kubectl create -f zookeeper.json```
+```kubectl create -f v1beta3/zookeeper.json```
```
Comment (nikhiljindal) [inline examples/storm/README.md]: These changes in this file are not required anymore.
Please revert.

**Comment by nikhiljindal** [comment]
LGTM, modulo one comment

**Comment by bcbroussard** [comment]
@nikhiljindal fixed

**Comment by nikhiljindal** [comment]
Thanks! Lgtm.

**Pull request merged**